app.rb

```diff
@@ -17,10 +17,25 @@ class GovukDependencies < Sinatra::Base
   get '/team' do
     ungrouped_pull_requests = UseCases::FetchPullRequests.new.execute
     teams = UseCases::FetchTeams.new.execute
-    @pull_requests_by_team = Presenters::PullRequestsByTeam.new.execute(
+    pull_requests_by_team = Presenters::PullRequestsByTeam.new.execute(
       teams: teams,
       ungrouped_pull_requests: ungrouped_pull_requests
     )
-    erb :team
+    erb :team, locals: { pull_requests_by_team: pull_requests_by_team }
+  end
+
+  get '/team/:team_name' do
+    ungrouped_pull_requests = UseCases::FetchPullRequests.new.execute
+    teams = UseCases::FetchTeams.new.execute
+    grouped_pull_requests = Presenters::PullRequestsByTeam.new.execute(
+      teams: teams,
+      ungrouped_pull_requests: ungrouped_pull_requests
+    )
+
+    pull_requests_for_team = grouped_pull_requests.select do |team|
+      team.fetch(:team_name) == '#' + params.fetch(:team_name)
```

Comment:
I think the team name is always a slack channel:

https://docs.publishing.service.gov.uk/apps.json

Reply:
Yeah - we add the hash to the URL parameter so that it matches what comes out of apps.json as they're grouped by the team_names from there. E.g. `/team/email` will match `#email` from the apps.json team name

Reply:
Urgh, totally misread the code. Thought this was coming from the API. Move along!